style.css

```diff
@@ -412,3 +412,158 @@ textarea {
   background-color: #6070ff;
   color: #fff;
 }
+
+@media (min-width: 768px) {
+  .main-div {
+    border-radius: 0 0 0 150px;
+  }
+
+  .header {
+    padding: 25px 12%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: #6070ff;
+  }
+
+  .info-container {
+    padding: 225px 17%;
+    background-image: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .greetings {
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    color: #172b4d;
+    margin-bottom: 15px;
+  }
+
+  .me {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
```

Comment:
- [optional ] kindly, use meaning full class names to make your code understandable  for others 😘😂